Bump version 3.0.6

q231950 opened the pull request:
This pull request bumps the version of the library to 3.0.6.

__Issue:__ <Github Issue>

__Dependencies:__
- [ ] Unit Tests

Commit:
Update readme

This reverts commit d3881391a7ca9d61f3d274c5bc08116383597143.

Commit:
Bump version to 3.0.6

This reverts commit 666a5467e4e78482967241ee6b69943aedd93427.

MyDogTom commented on gradle.properties:
```diff
@@ -1,6 +1,6 @@
-VERSION=3.0.5
-VERSION_CODE=14
+VERSION=3.0.6
+VERSION_CODE=1
```
I'm not sure, what the usage/meaning of `version_code`. Perhaps, it should increase all the time?

q231950 replied:
As I understand it's the build number of that version. In that case it would be correct to reset it per version bump.

q231950 replied:
Reading some [documentation](https://developer.android.com/studio/publish/versioning) I think you are right @MyDogTom . 

> This number is used only to determine whether one version is more recent than another, with higher numbers indicating more recent versions

I will set it to `15`

Commit:
Correct version code
The [docs](https://developer.android.com/studio/publish/versioning) tell that the higher the version code, the newer the version:
`monotonically increase the value with each release`
